src/javascript/JContent/JContent.accordion-items.jsx

```diff
@@ -80,10 +83,16 @@ export const jContentAccordionItems = registry => {
         label: 'jcontent:label.contentManager.navigation.pages',
         defaultPath: siteKey => '/sites/' + siteKey,
         getPathForItem: node => {
-            const pages = node.ancestors.filter(n => n.primaryNodeType.name === 'jnt:page');
+            const pages = node.ancestors
+                .filter(n => n.primaryNodeType.name === 'jnt:page');
             return pages[pages.length - 1].path;
         },
-        canDisplayItem: node => !filesRegex.test(node.path) && !contentsRegex.test(node.path),
+        canDisplayItem: ({selectionNode, folderNode}) =>
+            selectionNode ? !filesRegex.test(selectionNode.path) &&
+                !contentsRegex.test(selectionNode.path) :
```

Comment:
maybe also check everythingUnderSitesRegex for the selectionNode
```suggestion
            selectionNode ? everythingUnderSitesRegex.test(selectionNode.path) && 
                !filesRegex.test(selectionNode.path) &&
                !contentsRegex.test(selectionNode.path) :
```